pkg/actions/mustgather.go

```diff
@@ -0,0 +1,364 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package actions
+
+import (
+	"archive/tar"
+	"archive/zip"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/eclipse/codewind-installer/pkg/appconstants"
+	"github.com/eclipse/codewind-installer/pkg/docker"
+	"github.com/eclipse/codewind-installer/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var codewindHome = filepath.Join(homeDir, ".codewind")
+var nowTime = time.Now().Format("20060102150405")
```

Comment:
I was going to comment on this constant. Then I learnt about go's "magical reference date"...

Reply:
Crazy isn't it!